diff --git a/node_modules/@nestjs-query/core/dist/src/common/deep-partial.type.d.ts b/node_modules/@nestjs-query/core/dist/src/common/deep-partial.type.d.ts
index c0f27ef..7e816f3 100644
--- a/node_modules/@nestjs-query/core/dist/src/common/deep-partial.type.d.ts
+++ b/node_modules/@nestjs-query/core/dist/src/common/deep-partial.type.d.ts
@@ -1,6 +1,4 @@
 /**
  * Create a Partial that will allow for partials of nested objects and arrays.
  */
-export declare type DeepPartial<T> = {
-    [P in keyof T]?: T[P] extends Array<infer U> ? Array<DeepPartial<U>> : T[P] extends ReadonlyArray<infer U2> ? ReadonlyArray<DeepPartial<U2>> : DeepPartial<T[P]>;
-};
+export declare type DeepPartial<T> = Partial<T>;
