diff --git a/node_modules/pipedream/components/google_sheets/utils.mjs b/node_modules/pipedream/components/google_sheets/utils.mjs
index a8c4216..c6caf28 100644
--- a/node_modules/pipedream/components/google_sheets/utils.mjs
+++ b/node_modules/pipedream/components/google_sheets/utils.mjs
@@ -7,7 +7,6 @@
  * omitEmptykey({ "": "bar" });
  *
  * @param {object} object the source object
- * @param {...*} [object.omittedObj] the properties of `object` whose keys are not empty strings
  * @returns {object} the new object
  */
 function omitEmptyKey({
diff --git a/node_modules/pipedream/components/notion/actions/create-page-from-database/create-page-from-database.mjs b/node_modules/pipedream/components/notion/actions/create-page-from-database/create-page-from-database.mjs
index 92a8956..8ab8f2c 100644
--- a/node_modules/pipedream/components/notion/actions/create-page-from-database/create-page-from-database.mjs
+++ b/node_modules/pipedream/components/notion/actions/create-page-from-database/create-page-from-database.mjs
@@ -1,4 +1,4 @@
-import { pick } from "lodash-es";
+import { pick } from "lodash";
 import notion from "../../notion.app.mjs";
 import base from "../common/base-page-builder.mjs";
 
diff --git a/node_modules/pipedream/components/notion/actions/update-page/update-page.mjs b/node_modules/pipedream/components/notion/actions/update-page/update-page.mjs
index 550f513..6cf71cb 100644
--- a/node_modules/pipedream/components/notion/actions/update-page/update-page.mjs
+++ b/node_modules/pipedream/components/notion/actions/update-page/update-page.mjs
@@ -1,6 +1,6 @@
 import notion from "../../notion.app.mjs";
 import base from "../common/base-page-builder.mjs";
-import { pick } from "lodash-es";
+import { pick } from "lodash";
 
 export default {
   ...base,
