diff --git a/node_modules/pipedream/components/google_sheets/utils.mjs b/node_modules/pipedream/components/google_sheets/utils.mjs
index a8c4216..c6caf28 100644
--- a/node_modules/pipedream/components/google_sheets/utils.mjs
+++ b/node_modules/pipedream/components/google_sheets/utils.mjs
@@ -7,7 +7,6 @@
  * omitEmptykey({ "": "bar" });
  *
  * @param {object} object the source object
- * @param {...*} [object.omittedObj] the properties of `object` whose keys are not empty strings
  * @returns {object} the new object
  */
 function omitEmptyKey({
